.circleci/css-size-report/sizeReport.js

```diff
@@ -0,0 +1,118 @@
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
+const Octokit = require('@octokit/rest');
+
+const octokit = new Octokit({ auth: process.env.GH_PR_TOKEN });
+
+const owner = process.env.CIRCLE_PROJECT_USERNAME; // patternfly
+const repo = process.env.CIRCLE_PROJECT_REPONAME;
+const prnum = process.env.CIRCLE_PR_NUMBER;
+
+const exitCode = 0;
+const results = {};
+// size of the previous version cee file being tested
+let prevResult;
+// percentage difference of css files
+let diff;
+// the number of files that have changed
+let totalFiles = 0;
+// the color of the result size value.
+let sizeCol = 'green';
+
+let html = '<table>';
+html += '<tr>';
+html += `<td>Name</td>`;
+html += `<td>Current(kb)</td>`;
+html += `<td>Previous(kb)</td>`;
+html += `<td>Diff %</td>`;
+html += '</tr>';
+
+// @todo add: 'dist/patternfly.css', 'dist/patternfly.min.css'
+glob
+  .sync('../../dist/components/**/*.css')
```

Comment:
We can we include layouts too? We have one duplicate (Flex layout vs Flex component), so you'd have to handle that special case in the HTML table.

Reply:
DOne